sql/plsql/PlSqlLexer.g4

```diff
@@ -2379,9 +2380,11 @@ PERIOD:         '.';
 UNSIGNED_INTEGER:    [0-9]+;
 APPROXIMATE_NUM_LIT: FLOAT_FRAGMENT ('E' ('+'|'-')? (FLOAT_FRAGMENT | [0-9]+))? ('D' | 'F')?;
 
+
 // Rule #--- <CHAR_STRING> is a base for Rule #065 <char_string_lit> , it incorporates <character_representation>
 // and a superfluous subtoken typecasting of the "QUOTE"
-CHAR_STRING: '\''  (~('\'' | '\r' | '\n') | '\'' '\'' | NEWLINE)* '\'';
+CHAR_STRING: CHAR_STRING_FRAGMENT;
+fragment CHAR_STRING_FRAGMENT: '\''  (~('\'' | '\r' | '\n') | '\'' '\'' | NEWLINE)* '\'';
```

Comment:
Now I see `CHAR_STRING_FRAGMENT` is used only in one use-site that's why it's not needed. `CHAR_STRING` can be restored to the previous version:

```suggestion
CHAR_STRING: '\''  (~('\'' | '\r' | '\n') | '\'' '\'' | NEWLINE)* '\'';
```